release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1089,6 +1089,13 @@ With this release, specifying an unhealthy pod eviction policy for pod disruptio
 
 For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
 
+[id="ocp-4-14-nodes-cgroupv2-default"]
+==== Linux Control Groups version 2 is now default
+
+Beginning with {product-title} 4.14, new installs will use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have install born dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the CgroupMode of the Node Config object to 'v1'.
```

Comment:
Can "install born dates" just be "installation dates" or "initial installation dates"? Or is that a specific term we need to use.
```suggestion
Beginning with {product-title} 4.14, new installs will use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the CgroupMode of the Node Config object to 'v1'.
```

Reply:
✓